.github/workflows/build-styleguide.yaml

```diff
@@ -0,0 +1,83 @@
+name: Build styleguide
```

Comment:
would be nice to move thi saction to the https://github.com/sulu/sulu-javascript-docs repo 🙂  i think we can use the `schedule` and the `workflow_dispatch` tigger: 
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events